b2luigi/__init__.py

```diff
@@ -68,3 +69,85 @@ def requires(_self):
         task_that_requires.requires = requires
 
         return task_that_requires
+
+
+class inherits(object):
+
+    """
+    Similar to `b2luigi.requires`, this copies the luigi.inherits functionality but allows specifying parameters you
```

Comment:
I understand your idea here, but not everyone reads the documentation from top to bottom and the reference to requires might even confuse people (because of course inherits does not do the same than requires). I think you can remove it here and maybe create a "See also" section (or alike) further down.

Reply:
Ok, done. However I noticed that the documentation from `b2luigi.inherits` doesn't show up in sphinx and neither does the documentation I added. Where should this show up do you think?